pom.xml

```diff
@@ -131,7 +131,7 @@
         <mysql.version>8.0.23</mysql.version>
         <osgi.version>4.2.0</osgi.version>
         <pax.web.version>7.1.0</pax.web.version>
-        <postgresql.version>42.2.8</postgresql.version>
+        <postgresql.version>42.2.13</postgresql.version>
```

Comment:
EAP7.4 certifies with jdbc driver version 42.2.18, is there any specific reason why we're upgrading to 42.2.13? Could we upgrade to 42.2.18 instead?

Reply:
Thanks @vramik, there's no specific reason, so I updated the PR to the version you suggested.